src/corehaptics.cs

```diff
@@ -403,4 +409,20 @@ interface CHHapticPattern {
 		[return: NullAllowed]
 		CHHapticPatternDefinition Export ([NullAllowed] out NSError outError);
 	}
-}
\ No newline at end of file
+
+	[Static]
+	[Internal]
+	[Mac (12,0), iOS (15,0), TV (15,0), MacCatalyst (15,0), NoWatch]
+	partial interface CHHapticAudioResourceKeys {
+		[Field ("CHHapticAudioResourceKeyUseVolumeEnvelope")]
+		NSString UseVolumeEnvelopeKey { get; }
+	}
+
+	[Mac (12,0), iOS (15,0), TV (15,0), MacCatalyst (15,0), NoWatch]
+	[StrongDictionary ("CHHapticAudioResourceKeys")]
+	partial interface CHHapticAudioResourceDefinition {
+		[Advice ("The default value is true.")]
+		bool UseVolumeEnvelope { get; set; }
```

Comment:
same as above, better use `bool?` to let developers not change the default

Reply:
So this is backed by a NSDictionary, AFAIK if they do nothing, the key should not be added. But We can do a bool?

Reply:
error BI1033: bgen: Limitation: can not automatically create strongly typed dictionary for (System.Nullable`1[System.Boolean]) the value type of the CoreHaptics.CHHapticPatternDefinition.EventWaveformUseVolumeEnvelope property

Reply:
hmm... is that not a `NSNumber` backed bool ?

Reply:
@dalexsoto why are we not able to do nullable? Do we need to fix that?

Reply:
Please leave your message after the tone. beep... Yeah [docs](https://docs.microsoft.com/en-us/xamarin/cross-platform/macios/binding/binding-types-reference#strong-dictionary-types) says nothing about nullable types so probably are not supported.

Reply:
Already supported as per the API diff:
```csharp
public class CHHapticAudioResourceDefinition : Foundation.DictionaryContainer {
	// constructors
	public CHHapticAudioResourceDefinition ();
	public CHHapticAudioResourceDefinition (Foundation.NSDictionary dictionary);
	// properties
	public bool? UseVolumeEnvelope { get; set; }
}
```